script/munin-run

```diff
@@ -44,31 +43,29 @@ my $paranoia = 0;
 
 my $config = Munin::Node::Config->instance();
 
-sub main
-{
-    my ($plugin, $arg) = parse_args();
+sub main {
+    my ( $plugin, $arg ) = parse_args();
 
     # Loads the settings from munin-node.conf.
     # Ensures that, where options can be set both in the config and in
     # @ARGV, the latter takes precedence.
     $paranoia = $config->{paranoia};
 
-    my $config = Munin::Node::Config->instance();
     $config->parse_config_from_file($conffile);
 
-    $services = Munin::Node::Service->new(
+    my $services = Munin::Node::Service->new(
         servicedir => $servicedir,
         defuser    => $config->{defuser},
         defgroup   => $config->{defgroup},
         pidebug    => $PIDEBUG,
     );
 
-    $config->reinitialize({
-        %$config,
-        paranoia   => $paranoia,
-    });
+    $config->reinitialize( {
+        %{$config},    ## no critic qw(ValuesAndExpressions::ProhibitCommaSeparatedStatements)
```

Comment:
Just wild guessing: is this expression supposed to merge the `config` hash together with the additional hash value and supply this new hash as an argument to the `reinitialize` method?
Maybe there is a better way to achieve this without overriding the policy?

Reply:
Yeah, that would be better, but I did not yet find a way, input welcome.

Reply:
Using a temporary var is *always* possible, and usually even preferred

Reply:
@cgzones: would you try to work around this (by using a separate variable?) one in order to save the warning override?

Reply:
I do not see a way to use a temporary variable :-(

Reply:
@steveschnepp: I think, it is time to prove your *a temporary var is always possible* claim :)